### PR TITLE
refactoring.

### PR DESCRIPTION
**Pull request by wasuken**
Refactoring to Slightly modified.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -44,8 +37,9 @@ mod tests {
 
     #[test]
     fn shuffle1() {
-        let mut x = vec![1, 2, 3];
+        let x = vec![1, 2, 3];
         let y = shuffle(x);
+		println!("{}", y.iter().fold(0, |sum, a| sum + a));
```
Comment (maetaro) [inline src/main.rs:42]: i think this line is unnecessary.

Comment (wasuken) [reply src/main.rs:42]: I'm sorry... I'm Scrap...

Comment (maetaro) [reply src/main.rs]: no. you are fine.

Comment (maetaro) [reply src/main.rs]: Hate code and don't hate people.

Comment (wasuken) [reply src/main.rs]: Refactoring!

**Pull request merged**